libiocage/lib/helpers.py

```diff
@@ -103,6 +103,79 @@ def validate_name(name):
     return bool(validate.fullmatch(name))
 
 
+# data -> bool | default
+def parse_bool(data, default=False):
+    """
+    try to parse booleans from strings
+
+    On success, it returns the parsed boolean
+    on failure it returns the `default`.
+    By default, `default` is `False`.
+
+    >>> parse_bool("YES")
+    True
+    >>> parse_bool("false")
+    False
+    >>> parse_bool("/opt/android-studio/build.txt")
+    False
```

Comment:
A method called parse_bool should only parse booleans or raise. If it can read a boolean-value from the mapping table, that's fine. `None` is okay too, but everything else should raise.

Instead of giving the `parse_bool` method a `default` argument, you could use a try-block in `parse_user_input` that returns the input data in case of failure.